src/global_scheduler/global_scheduler.c

```diff
@@ -136,19 +144,27 @@ void process_new_db_client(db_client_id db_client_id,
     plasma_photon_entry->aux_address = strdup(aux_address);
     plasma_photon_entry->photon_db_client_id = db_client_id;
     HASH_ADD_KEYPTR(
-        hh, state->plasma_photon_map, plasma_photon_entry->aux_address,
+        plasma_photon_hh, state->plasma_photon_map, plasma_photon_entry->aux_address,
         strlen(plasma_photon_entry->aux_address), plasma_photon_entry);
 
+    /* Add photon_db_client_id -> plasma_manager ip:port association to state.*/
+    HASH_ADD(photon_plasma_hh, state->photon_plasma_map,
+             photon_db_client_id, /* Key is the field name of entry struct. */
```

Comment:
I know, but this is a rather inconvenient stylistic choice. When a comment is on the same line as the code it comments on, there's no ambiguity which line of code it corresponds to.  When the comment is on its own line, it could correspond to the line before or the line after. It's only by convention that you know it's the latter.
It makes sense for header files, when we expect doxygen to generate API documentation, but why have the same constraint in the rest of the code, where comments are rather for the developers reading the code?